src/store/cache-store.ts

```
@@ -0,0 +1,114 @@
+
+    const entry = get().attestations[address];
+
+    if (!entry) {
+      if (appConfig.TESTNET) {
+        console.log(`log(getUserAttestations): No cached attestations found for address: ${address}`);
+      }
+
+      return null;
+    }
+
+    let isExpired: boolean = true;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)